### PR TITLE
Second homework

### PR DESCRIPTION
**Pull request by ryvik13**

### CODE REVIEW

**Comment by vladyslav-yermolin-moc** [comment]
Please merge to you branch fresh `main` branch and recreate your pr once again. We've added html and css linters. Sorry for the inconvenience.

File: homeworks/evgeny.tolchinsky_ryvik13/GamesProject/Style.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
+text {
```
[the remark this answers is not in the document]
Comment (ryvik13) [reply homeworks/evgeny.tolchinsky_ryvik13/GamesProject/Style.css]: @vladyslav-yermolin-moc fixed

File: homeworks/evgeny.tolchinsky_ryvik13/GamesProject/Style.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  font-family: roboto;
+  font-size: 22px;
+  font-weight: 400;
+  line-height: 32px;
+  text-align: left;
+  color: #EF4934;
+}
+
+p {
+  ont-family: roboto;
```
[the remark this answers is not in the document]
Comment (ryvik13) [reply homeworks/evgeny.tolchinsky_ryvik13/GamesProject/Style.css]: @vladyslav-yermolin-moc fixed

File: homeworks/evgeny.tolchinsky_ryvik13/GamesProject/index.html [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+                About me
+            </h1>
+            <p>
+                Hi! My name is Evgeny and I'm a Junior Frontend Developer.
+                I am already familiar with main Web Technologies like HTML, CSS, JavaScript and Git version control system.
+            </p>
+            <p>
+                This page was developed during the course
+            </p>
+            <p>
```
[the remark this answers is not in the document]
Comment (ryvik13) [reply homeworks/evgeny.tolchinsky_ryvik13/GamesProject/index.html]: @vladyslav-yermolin-moc fixed

**Review by vladyslav-yermolin-moc — APPROVED** [review]

**Pull request merged**